### DISCUSSION TITLE
audion seeker not working

### DISCUSSION THREAD

**Discussion by akhil-221** [bluefireteam/audioplayers#1211] [Q&A]
Hi,
I tried out the package and works perfectly but seeker is not working in android devices but its working in IOS simulators.

**Comment by rovshan-b** [comment]
I also tried in android, seeking works but does not lead to correct position. 
There was a similar problem in just_audio plugin, they fixed it by using this code:
```
extractorsFactory.setMp3ExtractorFlags(com.google.android.exoplayer2.extractor.mp3.Mp3Extractor.FLAG_ENABLE_INDEX_SEEKING);
```
I do not know if it is possible to make same adjustment here.

**Reply by Gustl22** [reply]
We don't use exoplayer yet. All that is waiting in #1691 to be done. But we are still depending on [Flutter](https://github.com/flutter/flutter/pull/137040) to get there.
